WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -19,16 +17,31 @@ class AccountSettingsService {
         static let pollingInterval = 60.0
     }
 
+    enum Notifications {
+        static let settingsChanged = "AccountSettingsServiceSettingsChanged"
+        static let refreshStatusChanged = "AccountSettingsServiceRefreshStatusChanged"
```

Comment:
Side comment (sorry, just saw that on the third review!).

Just to maintain (simmetry?) maybe...

```
static let settingsChanged
static let statusChanged
```

Or

```
static let accountSettingsChanged
static let refreshStatusChanged
```

Otherwise feels a bit odd (?)